src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/Reviews/Reviews.types.ts

```diff
@@ -1,16 +1,12 @@
-import { AssessmentId } from 'modules/Dashboard/api';
-
 import { ReviewData } from '../FeedbackReviews.types';
 
-export type OnReviewerAnswerRemove = ({ assessmentId }: AssessmentId) => Promise<void>;
-
 export type ReviewsProps = {
   isLoading: boolean;
-  reviewsError: string | null;
-  reviewersData: ReviewData[];
-  removeReviewError: string | null;
-  removeReviewLoading: boolean;
-  onReviewerAnswersRemove: OnReviewerAnswerRemove;
+  reviewError: string | null;
+  reviewerData: ReviewData[];
+  removeReviewsError: string | null;
+  removeReviewsLoading: boolean;
+  onReviewerAnswersRemove: (assessmentId: string) => Promise<void>;
```

Comment:
```suggestion
  onReviewerAnswersRemove: (reviewId: string) => Promise<void>;
```
Why not to reuse this type in child components as `OnReviewerAsnwersRemove`?